code/modules/projectiles/bullet.dm

```diff
@@ -738,6 +738,129 @@ toxic - poisons
 			take_bleeding_damage(M, proj.shooter, 3, DAMAGE_CUT, 1)
 		..()
 
+/datum/projectile/bullet/kuvalda_slug //engine block destroying slug. not as fun as Buck, but longer range and AP.
+	icon_state = "4gauge"
+	hit_ground_chance = 66
+	implanted = null
+	damage = 65
+	armor_ignored = 0.8
+	stun = 10
+	hit_type = DAMAGE_STAB
+	dissipation_rate = 10
+	shot_sound = 'sound/weapons/kuvalda.ogg'
+	dissipation_delay = 6
+	casing = /obj/item/casing/shotgun/gray
+	var/image/blood_image
+	var/bleeding = FALSE // if we're spitting blood
+	var/mob/living/unfortunate  // who was last hit, by this bullet
+	var/turf/last_projectile_loc //1:many data:projectile pairing
+	var/turf/last_turf
+	on_launch(obj/projectile/O)
+		O.AddComponent(/datum/component/gaseous_projectile)
```

Comment:
use of gaseous_projectile here feels incorrect - this will cause unexpected behaviour e.g.: bullets being blocked by atmos forcefields. Find or make a better-fitting solution